src/ray/common/grpc_util.h

```diff
@@ -149,4 +150,16 @@ inline absl::flat_hash_map<K, V> MapFromProtobuf(
   return absl::flat_hash_map<K, V>(pb_map.begin(), pb_map.end());
 }
 
+inline grpc::ChannelArguments CreateDefaultChannelArguments() {
+  grpc::ChannelArguments arguments;
+  if (::RayConfig::instance().grpc_client_keepalive_time_ms() > 0) {
+    arguments.SetInt(GRPC_ARG_KEEPALIVE_TIME_MS,
+                     ::RayConfig::instance().grpc_client_keepalive_time_ms());
+    arguments.SetInt(GRPC_ARG_KEEPALIVE_TIMEOUT_MS,
+                     ::RayConfig::instance().grpc_client_keepalive_timeout_ms());
+    arguments.SetInt(GRPC_ARG_KEEPALIVE_PERMIT_WITHOUT_CALLS, 1);
```

Comment:
also should I set this to 0?

Reply:
yeah it is 0 by default. I think it is okay to set 0. This means we don't send a keepalive if we don't have a in-flight request